samples/snippets/list_testcase_results.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START dialogflow_list_test_case_results_sample]
+
+from google.cloud.dialogflowcx_v3.services.test_cases.client import TestCasesClient
+from google.cloud.dialogflowcx_v3.types.test_case import ListTestCaseResultsRequest
+
+
+def list_test_case(project_id, agent_id, test_id, location):
+
+    req = ListTestCaseResultsRequest()
+    req.parent = (
```

Comment:
Please use an f-string for better readability:

```py
req.parent = f"projects/{project_id}/locations/{location}/agents/{agent_id}/testCases/{test_id}"
```

Alternatively, you can use the path helper provided on the client.

https://github.com/googleapis/python-dialogflow-cx/blob/dd2b2fd345e91fa3da1290f6896ac6b521c2eb7b/google/cloud/dialogflowcx_v3beta1/services/test_cases/client.py#L267-L271

```py
req.parent = TestCasesClient.test_case_path(project=project_id, location=location, agent=agent, test_case=test_id)
```